src/Fusion.Summary.Functions/Functions/DepartmentResourceOwnerSync.cs

```diff
@@ -52,6 +53,7 @@ public async Task RunAsync(
     {
         _serviceBusConnectionString = configuration["AzureWebJobsServiceBus"];
         _weeklySummaryQueueName = configuration["department_summary_weekly_queue"];
+        _departmentFilter = configuration["departmentFilter"]?.Split(',', StringSplitOptions.RemoveEmptyEntries) ?? [];
```

Comment:
Do we need to clean up white spaces?

Reply:
Maybe something like this would be better? 
```suggestion
        _departmentFilter = configuration["departmentFilter"]?.Split(',').Where(f => !string.IsNullOrWhiteSpace(f)).ToArray() ?? [];
```

If one accidentally adds an extra ', ' it will be removed. Trying to safeguard against accidentally including almost every department due to an empty white space. Maybe adding PRD to the null fallback could also be good idea

Reply:
Pushed a change, now it will also trim values. Think I misunderstood how the department strings work